portal/server/routes/middleware/validateSqlId/index.ts

```diff
@@ -0,0 +1,13 @@
+import { NextFunction, Request, Response } from 'express';
+import { isValidSqlId } from '../../../validation/validate-ids';
+
+export const validateSqlId = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Not really sure which of these I prefer, but another way to define middlewares in typescript is to use the `RequestHandler` type:
```typescript
import { RequestHandler } from 'express';
...

export const validateSqlId: RequestHandler = (req, res, next) => { ... };
```
I don't think it needs to be changed here, but just something interesting that we might want to start doing 🤷